ocs_ci/ocs/resources/mcg.py

```diff
@@ -587,7 +587,7 @@ def _check_state():
             return False
 
         try:
-            for reached_state in TimeoutSampler(180, 10, _check_state):
+            for reached_state in TimeoutSampler(600, 40, _check_state):
```

Comment:
3 minutes changed to 10 minutes.
why?

The reason I'm asking is to ensure we don't have a regression in performance of reaching the desired state

Reply:
The change is because we block and enable IO on the S3 bucket by using BucketPolicies. Even after drilling down the NB code and verifying that it refreshes in the intervals it should, we noticed it sometimes just takes a very long time for the policy to apply, so basically - waiting for AWS.

Reply:
@MeridianExplorer is this PR still needed then?

Reply:
@ebenahar - Yep, there are still two changes for other timings in the MCG tests. Just needs further research and testing. There's also a bug regarding this specific part (backingstore refuses to update its state) - waiting for a response from NB

Reply:
ack